data/json/professions.json

```diff
@@ -3379,5 +3379,37 @@
                 "hardtack"
             ]
         }
+    },{
+
+        "type": "profession",
+        "ident": "drone_op",
+        "name": "Drone Operator",
+        "description": "You had a job programming machines like automatic street cleaners, newsbots and pizza delivery drones. Now the all the drones carry guns instead of pizza.",
```

Comment:
I'd say that it could work like it works right now: drones deliver some of the stuff, people deliver the rest.
Not every chain can afford to use drones all the time, range problems, noise etc.
Just like how police drones work with police people rather than replacing them.